.semaphore/push-images/node.yml

```diff
@@ -26,18 +26,15 @@ global_job_config:
       - export BRANCH_NAME=$SEMAPHORE_GIT_BRANCH
 
 blocks:
-  - name: "Publish node images"
+  - name: "Publish node images & manifests"
     dependencies: []
     skip:
       when: "branch !~ '.+'"
     task:
       jobs:
-        - name: Linux multi-arch
+        - name: non-arm64
           commands:
             - if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then make -C node cd EXCLUDEARCH=arm64 CONFIRM=true; fi
-        - name: Windows
-          commands:
-            - if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then make -C node release-windows CONFIRM=true; fi
```

Comment:
will this mean we won't build windows images anymore? is this covered elsewhere?